importer/api/__init__.py

```diff
@@ -74,7 +74,7 @@ def import_course_from_path(path, repo_id, user_id):
     """
     bundle = XBundle()
     bundle.import_from_directory(path)
-    return import_course(bundle, repo_id, user_id)
+    import_course(bundle, repo_id, user_id)
```

Comment:
It was just cleaning as I went along. The called function doesn't return a value, so it was just misleading. Is there any reason _not_ to change it here?